core/logger/tailer.go

```diff
@@ -36,6 +36,7 @@ type LogTailerParams struct {
 	StartTime     time.Time
 	MinLevel      loggo.Level
 	InitialLines  int
+	Limit         int
```

Comment:
InitialLines and Limit are essentially the same thing - one tails and the other doesn't. I don't think we want an extra param here.

Reply:
Hi Ian but for 3.6 we are still keeping both right? InitialLines value comes from -n and --lines value but Limit value comes from --limit value. Do you mean to change the below "limit" name to "n" and "lines" instead?

	f.UintVar(&c.params.Backlog, "n", defaultLineCount, "Show this many of the most recent lines and continue to append new ones")
	f.UintVar(&c.params.Backlog, "lines", defaultLineCount, "")
	f.UintVar(&c.params.Limit, "limit", 0, "Show this many of the most recent lines and then exits")

Reply:
Resolved offline, thanks Ian